README.md

```diff
@@ -69,6 +69,16 @@ brew tap anchore/syft
 brew install syft
 ```
 
+**Container Image**
```

Comment:
```suggestion
**Docker**
```

Reply:
Why do we want to say Docker instead of container image?

Reply:
I think container image is a better term as this will allow us to include other engines in the future.

Reply:
actually, on further consideration, I retract this...